### ISSUE TITLE
复现结果比论文低很多

### ISSUE THREAD

**Issue by hhr114** [BUPT-GAMMA/CPF#8]
你好，因为文中使用的cora等数据集的划分和标准划分不同，所以我修改了load data部分代码，teacher logits直接使用独立的GCNII模型的结果，teacher测试精度85%多，但最终student PLP的结果只有60%不到，比原文汇报的低很多，请问是什么原因呢？

部分修改如下：
![image.png](https://ae03.alicdn.com/kf/Haeaa191450c24ba29595de2c4031cd51N.png)
关于teacher logits，我也对load_cascades相应进行了修改，直接由 GCNII 的logits经softmax后作为cas。
最终结果截图（teacher 85%，但student 不到60%）——
![image.png](https://ae01.alicdn.com/kf/H44f768e130a04ff1a5a3615f3519cae8d.png)

**Comment by J-cabin** [comment]
目前没有收到类似的反馈，建议您使用与论文一致的数据集、数据划分进行实验。
